src/views/PoEMap.vue

```diff
@@ -84,116 +80,149 @@ function onEachOddity(feature, layer) {
+            return markerAlias(latlng, { icon: grineerMarker });
           },
+          onEachFeature: onEachFeature,
         },
-        {
-          name: 'Oddity',
-          json: lorefish,
-          opts: {
-            pointToLayer: function(feature, latlng) {
-              return markerAlias(latlng, { icon: oddityMarker });
-            },
-            onEachFeature: onEachOddity,
+      },
+      {
+        name: 'Oddity',
+        json: lorefish,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: oddityMarker });
           },
+          onEachFeature: onEachOddity,
         },
-        {
-          name: 'Cetus Wisp',
-          json: wisp,
-          opts: {
-            pointToLayer: function(feature, latlng) {
-              return markerAlias(latlng, { icon: wispMarker });
-            },
-            onEachFeature: onEachFeature,
+      },
+      {
+        name: 'Cetus Wisp',
+        json: wisp,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: wispMarker });
           },
+          onEachFeature: onEachFeature,
         },
-        {
-          name: 'Vomvalyst Lure',
-          json: lure,
-          opts: {
-            pointToLayer: function(feature, latlng) {
-              return markerAlias(latlng, { icon: lureMarker });
-            },
-            onEachFeature: onEachFeature,
+      },
+      {
+        name: 'Vomvalyst Lure',
+        json: lure,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: lureMarker });
           },
+          onEachFeature: onEachFeature,
         },
-        {
-          name: 'Cave Entrance',
-          json: cave,
-          opts: {
-            pointToLayer: function(feature, latlng) {
-              return markerAlias(latlng, { icon: caveMarker });
-            },
-            onEachFeature: onEachFeature,
+      },
+      {
+        name: 'Cave Entrance',
+        json: cave,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: caveMarker });
           },
+          onEachFeature: onEachFeature,
         },
-      ],
-    };
-  },
+      },
+    ],
+  };
+}
+
+function defaultToggleValues() {
+  let defaults = {};
+  dataFn().geo.forEach((g) => {
+    defaults[g.name + '-toggle-value'] = true;
+  });
+  return defaults;
+}
+
+const markerAlias = L.marker;
+const labelAlias = L.circleMarker;
+
+export default {
+  name: 'Poemap',
+  data: dataFn,
   methods: {
     track() {
       this.$ga.page('/poe/map');
     },
   },
-  mounted: function() {},
+  mounted: function() {
+    this.map = this.$refs.pmap.mapObject;
+    // Get our toggle values from local storage
+    let toggles = localStorage.poeMapToggles ? JSON.parse(localStorage.poeMapToggles) : defaultToggleValues();
```

Comment:
could you push them to the state, keeping with how settings are normally loaded?